python/beeai_framework/agents/react/agent.py

```diff
@@ -158,7 +158,7 @@ async def _run(
                     )
                 )
 
-                for key in ["partialUpdate", "update"]:
+                for key in ["partial_update", "update"]:
```

Comment:
this change needs to be documented in the final commit message as a braking change

Reply:
this case is actually a bug since it was already snake_case everywhere else